### DISCUSSION TITLE
Cant Custom Error 'before' with global function now()

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [livewire/livewire#3951] [1. Help] [answered]
Hello everybody.

I have problem when customizing the error validation of 'before' with adding now() laravel global function but its get error exception 'Constant expression contains invalid operations'. This is my code

` protected function rules() {
        return [
            'unit' => ['required'],
            'date' => ['required', 'before:' . now()->addMonth()],
        ];
}`    

 `protected $message = [
        'date.before:' . now()->addMonth() => 'Date cannot contain next month'
    ];`

Is there any other way to get this code work and display the custom message?

**Comment by joshhanley** [comment, marked as answer]
@lutfifirdaus when defining the property in the messages array, you should only add the validation type `before` but not the details `now()->addMonth()`. See here in the Laravel docs https://laravel.com/docs/8.x/validation#specifying-a-custom-message-for-a-given-attribute

So it should look something like this

```php
protected $message = [ 'date.before' => 'Date cannot contain next month' ];
```

Hope this helps!

**Reply by lutfifirdaus** [reply]
Thanks for your reply.  its work!

**Reply by joshhanley** [reply]
No worries 🙂
